src/sensor_state_data/data.py

```diff
@@ -30,6 +35,8 @@ class SensorUpdate:
     devices: dict[str | None, SensorDeviceInfo]
     entity_descriptions: dict[DeviceKey, SensorDescription]
     entity_values: dict[DeviceKey, SensorValue]
+    binary_entity_descriptions: dict[DeviceKey, BinarySensorDescription]
+    binary_entity_values: dict[DeviceKey, BinarySensorValue]
```

Comment:
If we ever end up with a lot more platforms, we can add new platform descriptions/values under a dict instead and leave these in place until we migrate away from the top level ones